packages/strapi-plugin-content-manager/admin/src/containers/CollectionTypeWrapper/index.js

```diff
@@ -28,24 +29,41 @@ const CollectionTypeWrapper = ({ allLayoutData, children, from, slug }) => {
   const isCreatingEntry = id === 'create';
 
   const fetchURL = useMemo(() => {
-    if (isCreatingEntry) {
+    if (isCreatingEntry && !origin) {
       return null;
     }
 
-    return getRequestUrl(`${slug}/${id}`);
-  }, [slug, id, isCreatingEntry]);
+    return getRequestUrl(`${slug}/${origin || id}`);
+  }, [slug, id, isCreatingEntry, origin]);
+
+  const cleanClonedData = useCallback(
```

Comment:
I don't think we need the origin statement here if we apply the function only in the useEffect.

Reply:
I followed the eslint statement
`React Hook useMemo has a missing dependency: 'origin'. Either include it or remove the dependency array` even if I agree with you